docs/source/users/fit_tutorial.rst

```diff
@@ -38,121 +36,363 @@ We start by loading and processing data just as we did for the parameter inferen
+       r = 1  
+       center.0 = 1  
+       center.1 = 1  
+       center.2 = 1  
+    Iter       2    CHI-SQUARE =  1452264.089  DOF =  39997
+       r = 0.898518235  
+       center.0 = 1.003214606  
+       center.1 = 0.9978521719  
+       center.2 = 1.014361149  
+    Iter       3    CHI-SQUARE =  806449.818  DOF =  39997
+       r = 0.9073627025  
+       center.0 = 1.005863598  
+       center.1 = 0.9957062884  
+       center.2 = 1.045412833  
+    Iter       4    CHI-SQUARE =  485324.5589  DOF =  39997
+       r = 0.9452177325  
+       center.0 = 1.006911119  
+       center.1 = 0.9939173194  
+       center.2 = 1.07198883  
+    Iter       5    CHI-SQUARE =  420078.987  DOF =  39997
+       r = 0.9588134486  
+       center.0 = 1.006953804  
+       center.1 = 0.9931590131  
+       center.2 = 1.084210817  
+    Iter       6    CHI-SQUARE =  411628.2723  DOF =  39997
+       r = 0.9589424617  
+       center.0 = 1.006880613  
+       center.1 = 0.9929427182  
+       center.2 = 1.088112307  
+    Iter       7    CHI-SQUARE =  410494.0376  DOF =  39997
+       r = 0.9574525307  
+       center.0 = 1.006838214  
+       center.1 = 0.992886033  
+       center.2 = 1.089302588  
+    Iter       8    CHI-SQUARE =  410331.6001  DOF =  39997
+       r = 0.9565875508  
+       center.0 = 1.006819237  
+       center.1 = 0.9928712659  
+       center.2 = 1.089696833  
+    Iter       9    CHI-SQUARE =  410307.9575  DOF =  39997
+       r = 0.9562069239  
+       center.0 = 1.006811423  
+       center.1 = 0.9928673014  
+       center.2 = 1.089836875  
+    Iter      10    CHI-SQUARE =  410304.5094  DOF =  39997
+       r = 0.9560533245  
+       center.0 = 1.006808322  
+       center.1 = 0.9928661767  
+       center.2 = 1.08988857  
+    Iter      11    CHI-SQUARE =  410304.0062  DOF =  39997
+       r = 0.9559935222  
+       center.0 = 1.006807116  
+       center.1 = 0.9928658363  
+       center.2 = 1.089908104  
+    Iter      12    CHI-SQUARE =  410303.9332  DOF =  39997
+       r = 0.9559702453  
+       center.0 = 1.00680665  
+       center.1 = 0.9928657253  
+       center.2 = 1.089915414  
+    Iter      13    CHI-SQUARE =  410303.9225  DOF =  39997
+       r = 0.9559614218  
+       center.0 = 1.006806474  
+       center.1 = 0.9928656889  
+       center.2 = 1.089918235  
+    Iter      14    CHI-SQUARE =  410303.921  DOF =  39997
+       r = 0.9559580746  
+       center.0 = 1.006806407  
+       center.1 = 0.9928656759  
+       center.2 = 1.089919325  
+    Iter      15    CHI-SQUARE =  410303.9207  DOF =  39997
+       r = 0.9559566861  
+       center.0 = 1.006806381  
+       center.1 = 0.9928656724  
+       center.2 = 1.089919691  
+
+Here we have used an :class:`.ExactModel` which takes a function ``calc_func``
+to apply on the :class:`.Scatterer` (we have used :func:`.calc_holo` here).
+The :class:`.ExactModel` is the default for :func:`.fit` but HoloPy also

```

Comment:
My 2 cents is that the `fit` function should have defaults that are the most generally helpful. Since we currently need an alpha to fit holograms, I think that `fit` should default to `AlphaModel`, not `ExactModel`.

Reply:
The whole point is that the functional interface it's easy to use, so I made the change you suggested.